read-fonts/src/tables/variations.rs

```diff
@@ -806,7 +856,7 @@ pub struct TupleDeltaIter<'a, T> {
     points: Option<PackedPointNumbersIter<'a>>,
     next_point: usize,
     x_iter: DeltaRunIter<'a>,
-    y_iter: Option<Skip<DeltaRunIter<'a>>>,
+    y_iter: Option<DeltaRunIter<'a>>,
```

Comment:
Optional, feels a little weird that only y is an option, could it just be an iter that's empty instead of None?

Reply:
The iterator would need to produce an infinite stream of zeroes to fit into the current logic and I think that complication might lead to bugs.

I’ll replace this with an enum that holds two iters for points and one for scalars which should make this more clear.

Reply:
That rationale would make a fine comment :)